src/core/font_renderer.js

```diff
@@ -612,17 +637,28 @@ var FontRendererFactory = (function FontRendererFactoryClosure() {
       return fn;
     },
 
-    compileGlyph(code) {
+    compileGlyph(code, glyphId) {
       if (!code || code.length === 0 || code[0] === 14) {
         return noop;
       }
 
+      let fontMatrix = this.fontMatrix;
+      if (this.isCFFCIDFont) {
+        let fdIndex = this.fdSelect.getFDIndex(glyphId);
```

Comment:
Let's add a comment here about why we ignore the top dict's font matrix (similar to your comment in the PR).

Reply:
I added a comment.